app/build.gradle

```diff
@@ -246,7 +264,7 @@ dependencies {
     implementation 'com.google.android:flexbox:2.0.1'
 
     // PW
-    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Duplicated with line 174?

Reply:
Yes. It doesn't break anything to remove it.